src/Umbraco.Web.Common/Extensions/UmbracoCoreServiceCollectionExtensions.cs

```diff
@@ -293,10 +293,24 @@ public static IServiceCollection AddUmbracoSqlServerSupport(this IServiceCollect
         public static IServiceCollection AddUmbracoHostedServices(this IServiceCollection services)
         {
             services.AddHostedService<HealthCheckNotifier>();
+            services.AddHostedService<KeepAlive>();
+            services.AddHostedService<LogScrubber>();
+            services.AddHostedService<TempFileCleanup>();
 
             return services;
         }
```

Comment:
In theoy this could now be called from the `SchedulerComposer`. Not sure whether we should stick to the umbraco way or .net way here. Thoughts?

Reply:
No major preference, but arguably this work to use hosted services is already moving from a custom Umbraco way to a more "standard .NET Core" way, so arguably we should continue that to include registration.